docs/getting-started/proofs-and-verification/proofs.md

```diff
@@ -44,4 +47,54 @@ message GetDataProofResponse {
 message MPTrieProofElement {
   repeated bytes hashes = 1;
 }
-```
\ No newline at end of file
+```
+
+### Proofs
+
+#### Ledger connectivity proof
+Proof generation:
+- Zero step – user provided with receipt (block header) from another user who claim that some block is part of ledger
+- First step – user requests header of last block in ledger, without discovering what block he wants to validate
```

Comment:
Alice requests from the server the header of the last block in the ledger, without revealing to the server which block she wants to validate